6_Agent_Deployment/backend_agent_api/agent_api.py

```
@@ -493,6 +493,118 @@
+    
+    Args:
+        request: FileUploadRequest with file details
+        credentials: Authorization credentials
+        
+    Returns:
+        Success response with processing status
+    """
+    try:
+        # Validate authorization
+        if not credentials:
+            raise HTTPException(status_code=401, detail="Authorization required")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)